MinecraftClient/MinecraftClient.csproj

```diff
@@ -158,7 +160,7 @@
     <Compile Include="Mapping\EntityPalettes\EntityPaletteGenerator.cs" />
     <Compile Include="Mapping\EntityTypeExtensions.cs" />
     <Compile Include="Mapping\MaterialExtensions.cs" />
-    <Compile Include="MCLogger.cs" />
+    <Compile Include="Logger\FilteredLogger.cs" />

```

Comment:
For your information when you rename an existing file, it's better to use `git mv` as well so that the diff is visible instead of file creation + file deletion as in this pull request. You can do this this way:
1. Rename file in Visual Studio
2. Exit Visual Studio
3. Rename file back to the old name
4. git mv oldname.cs newname.cs
5. The name change is now applied in Git as well